element-hq/element-x-android
# Add 'send private read receipts' option in advanced settings

## Type of change


## Content

- Adds a 'read receipt' option to advanced settings so the client can save the last read message but won't share it with the other users in the rooms.

## Motivation and context

Closes #2204.

## Tests

- Go to Advanced settings, disable this option.
- Go to any room with unread messages where you have another account in the members. This should automatically mark the room as read.
- Get out of the room, check the room is read.
- From the other account, check the read receipt for your user didn't change in the timeline.

## Tested devices

- OS version(s): 14

## Checklist
